### PR TITLE
Added YMZC Network

### PR DESCRIPTION
**Pull request by Hikari0964**
### General:
* [X] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [X] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [X] No changes were made to the file's formatting.
* [X] There are no syntax errors.
* [X] There is no pre-existing mapping matching my id.
* [X] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [X] `id`: a lowercase string *(ex. `myserver`)*
  - [X] `name`: a string *(ex. `MyServerPvP`)*
  - [X] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:
* [X] My image is a `png` file.
* [X] I uploaded my image to the `logos` directory.
* [X] My image has a transparent background and is square (1:1 aspect ratio).
* [X] My image is `512` pixels in width and height.
* [X] My image's file name matches its mapping id.

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1370,5 +1370,12 @@
         "addresses": [
             "dropthem.eu"
         ]
+    },
+    {
+        "id": "ymzc",
+        "name": "YMZC Network",
+        "addresses": [
+            "mc.ymzc.pro"
```
Comment (jacobk999) [inline servers.json:1378]: Change it from `mc.ymzc.pro` to `ymzc.pro`

**Review by jacobk999 — DISMISSED** [review]

File: servers.json (hunk 1 above)
Comment (Hikari0964) [reply servers.json:1378]: ok

Comment (Hikari0964) [reply servers.json]: > Change it from `mc.ymzc.pro` to `ymzc.pro`

changed <3

**Review by imconnorngl — APPROVED** [review]

**Comment by Hikari0964** [comment]
fixed again <3
please view <3
thanks <3

**Review by imconnorngl — APPROVED** [review]

**Pull request merged**